corehq/apps/app_manager/tests/test_views.py

```diff
@@ -218,3 +220,33 @@ def test_default_new_app(self, mock):
         expected = '/apps/view/{}/?appcues=1'.format(app_id)    # Remove get param when APPCUES_AB_TEST is over
         self.assertTrue(redirect_location.endswith(expected))
         self.addCleanup(lambda: Application.get_db().delete_doc(app_id))
+
+    def test_get_apps_modules(self, mock):
+        self.app.add_module(Module.new_module("Module0", "en"))
+        self.app.save()
+
+        other_app = Application.new_app(self.domain.name, "OtherApp")
+        other_app.add_module(Module.new_module("Module0", "en"))
+        other_app.save()
+        self.addCleanup(lambda: Application.get_db().delete_doc(other_app.id))
+
+        linked_app = create_linked_app(self.domain.name, self.app.id, self.domain.name, 'LinkedApp')
```

Comment:
I think we have been trying to bring some consistency to using project for `Domain` object and `domain` for its name. If you can squeeze that change here for the tests in this file.